### PR TITLE
resolved leaderboard search bug on android

### PR DESCRIPTION
**Pull request by jackkranz**
FIx bug

### CODE REVIEW

File: Xamarin/SSW.Consulting/SSW.Consulting/Views/LeaderBoard.xaml.cs [hunk 1]
```diff
@@ -60,32 +63,41 @@ private void SearchButtonPress(object sender, EventArgs e)
...
+            else
                 {
                     searchFrame.FadeTo(1, 150);
                     searchFrame.TranslateTo(0, 0, 100);
                     leaderList.Margin = new Thickness(0, 60, 0, 0);
                     showBar = true;
-                }
-            
+                show = DateTime.Now;
+
```
Comment (AndreasLengkeek) [inline Xamarin/SSW.Consulting/SSW.Consulting/Views/LeaderBoard.xaml.cs]: @jackkranz make sure you're cleaning up empty lines at the end of methods for better code readability

File: Xamarin/SSW.Consulting/SSW.Consulting.Android/Properties/AndroidManifest.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="7" android:versionName="1.5" package="com.ssw.consulting" android:installLocation="auto">
-	<uses-sdk android:minSdkVersion="24" android:targetSdkVersion="28" />
+	<uses-sdk android:minSdkVersion="24" android:targetSdkVersion="27" />
```
Comment (AndreasLengkeek) [inline Xamarin/SSW.Consulting/SSW.Consulting.Android/Properties/AndroidManifest.xml]: Why are we lowering the target sdk version?

Comment (jackkranz) [reply Xamarin/SSW.Consulting/SSW.Consulting.Android/Properties/AndroidManifest.xml]: reverted

File: Xamarin/SSW.Consulting/SSW.Consulting/Views/LeaderBoard.xaml.cs (hunk 1 above)
Comment (jackkranz) [reply Xamarin/SSW.Consulting/SSW.Consulting/Views/LeaderBoard.xaml.cs]: noted

**Pull request merged**